src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Handjet:wght@300&family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <article class="card" data-qa="card">
+      <img
+        class="card__foto"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+      <h1 class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__star">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__reviews">Reviews: 5</span>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price1">Price:</p>
+        <p>$2,199</p>
+      </div>
```

Comment:
add more meaningful class names
```suggestion
      <div class="card__price">
        <p class="card__price-text">Price:</p>
        <p class="card__price-value">$2,199</p>
      </div>
```